src/models/generated/TModelHVDCLine.jl

```diff
@@ -0,0 +1,128 @@
+#=
+This file is auto-generated. Do not edit.
+=#
+
+#! format: off
+
+"""
+    mutable struct TModelHVDCLine <: DCBranch
+        name::String
+        available::Bool
+        active_power_flow::Float64
+        arc::Arc
+        r::Float64
+        l::Float64
+        c::Float64
+        active_power_limits_from::MinMax
+        active_power_limits_to::MinMax
+        services::Vector{Service}
+        ext::Dict{String, Any}
+        internal::InfrastructureSystemsInternal
+    end
+
+a HVDC T-Model DC line.
+
+# Arguments
+- `name::String`
+- `available::Bool`
+- `active_power_flow::Float64`
+- `arc::Arc`
+- `r::Float64`: Series Resistance system per-unit value
+- `l::Float64`: Series Inductance system per-unit value
+- `c::Float64`: Shunt capacitance system per-unit value
+- `active_power_limits_from::MinMax`
+- `active_power_limits_to::MinMax`
+- `services::Vector{Service}`: Services that this device contributes to
+- `ext::Dict{String, Any}`
+- `internal::InfrastructureSystemsInternal`: power system internal reference, do not modify
+"""
+mutable struct TModelHVDCLine <: DCBranch
```

Comment:
If this is a DCBranch, do we require that a DC branch connect two DC buses. As a result, TwoTerminalHVDC would imply an ACBranch 🤔

Reply:
This goes in the Arc change. We would need to have a DCArc and ACArc in this case. The generalization of using Union is just the straightforward approach.